.github/workflows/release.yml

```diff
@@ -268,9 +268,9 @@ jobs:
           RELEASE_COMMIT: "${{ github.sha }}"
         run: |
           # Write and read notes from a file to avoid quoting breaking things
-          echo "$ANNOUNCEMENT_BODY" > $RUNNER_TEMP/notes.txt
+          echo "$ANNOUNCEMENT_BODY" > "$RUNNER_TEMP/notes.txt"
```

Comment:
Isn't this file auto generated?

Reply:
Yeah, right. Not sure why one is quoted while the other isn't. I'll revert.